test/teiserver_web/controllers/api/admin/user_controller_test.exs

```diff
@@ -0,0 +1,405 @@
+      user = Central.Helpers.GeneralTestLib.make_user()
+
+      app =
+        OAuthFixtures.app_attrs(user.id)
+        |> Map.put(:uid, "test_app")
+        |> Map.put(:scopes, ["admin.map"])
+        |> OAuthFixtures.create_app()
+
+      token =
+        OAuthFixtures.token_attrs(user.id, app)
+        |> Map.put(:scopes, ["admin.map"])
+        |> OAuthFixtures.create_token()
+
+      conn =
+        conn
+        |> put_req_header("authorization", "Bearer #{token.value}")
+
+      user_data = %{
+        "name" => "testuser",
+        "email" => "test@example.com",
+        "password" => "testpassword123"
+      }
+
+      resp = conn |> post(create_user_path(), user_data) |> json_response(401)
+      assert resp["error"] =~ "unauthorized_client"
+    end
+  end
+
+  describe "edge cases" do
+    setup [:setup_user, :setup_generic_lobby_app, :setup_authed_conn]
+
+    test "handles very long email", %{authed_conn: conn} do
+      long_email = String.duplicate("a", 100) <> "@example.com"
+
+      user_data = %{
+        "name" => "testuser",
+        "email" => long_email,
+        "password" => "testpassword123"
+      }
+
+      resp = conn |> post(create_user_path(), user_data) |> json_response(200)
+      assert resp["user"]["email"] == long_email
+    end
+
+    test "handles special characters in name", %{authed_conn: conn} do
+      user_data = %{
+        "name" => "User-Name_123 [Test]",
+        "email" => "special@example.com",
+        "password" => "testpassword123"
+      }
+
+      resp = conn |> post(create_user_path(), user_data) |> json_response(200)
+      assert resp["user"]["name"] == "User-Name_123 [Test]"
+    end
+
+    test "handles unicode characters", %{authed_conn: conn} do
+      user_data = %{
+        "name" => "José María",
+        "email" => "josé@example.com",
+        "password" => "testpassword123"
+      }
+
+      resp = conn |> post(create_user_path(), user_data) |> json_response(200)
+      assert resp["user"]["name"] == "José María"
+      assert resp["user"]["email"] == "josé@example.com"
+    end
+
+    test "handles zero values in stats", %{authed_conn: conn} do
+      user_data = %{
+        "name" => "zerouser",
+        "email" => "zero@example.com",
+        "password" => "testpassword123",
+        "mu" => 0,
+        "sigma" => 0,
+        "play_time" => 0
+      }
+
+      resp = conn |> post(create_user_path(), user_data) |> json_response(200)
+      assert resp["user"]["name"] == "zerouser"
+    end
+
+    test "handles negative values in stats", %{authed_conn: conn} do
+      user_data = %{
+        "name" => "negativeuser",
+        "email" => "negative@example.com",
+        "password" => "testpassword123",
+        "mu" => -100,
+        "sigma" => -50,
+        "play_time" => -3600
+      }
+
+      resp = conn |> post(create_user_path(), user_data) |> json_response(200)
+      assert resp["user"]["name"] == "negativeuser"
+    end
```

Comment:
why did you add these 2 tests? They don't seem to be very useful to me but maybe I'm missing something.